app/models/user.rb

```diff
@@ -64,33 +64,33 @@ class User < ApplicationRecord
   validates :github_username, uniqueness: { allow_blank: true }
   validates :text_color_hex, format: /\A#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})\z/, allow_blank: true
   validates :bg_color_hex, format: /\A#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})\z/, allow_blank: true
-  validates :website_url, url: { allow_blank: true, no_local: true, schemes: ["https", "http"] }
+  validates :website_url, :employer_url, :mastodon_url,
+    url: { allow_blank: true, no_local: true, schemes: ["https", "http"] }
   # rubocop:disable Metrics/LineLength
   validates :facebook_url,
-              format: /\Ahttps:\/\/(www.facebook.com|facebook.com)\/[a-zA-Z0-9.]{5,50}\/?\Z/,
+              format: /\A(http(s)?:\/\/)?(www.facebook.com|facebook.com)\/.*\Z/,
               allow_blank: true
   validates :stackoverflow_url,
               allow_blank: true,
               format:
-              /\Ahttps:\/\/(www.stackoverflow.com|stackoverflow.com|www.stackexchange.com|stackexchange.com)\/([\S]{3,100})\Z/
+              /\A(http(s)?:\/\/)?(www.stackoverflow.com|stackoverflow.com|www.stackexchange.com|stackexchange.com)\/.*\Z/
   validates :behance_url,
               allow_blank: true,
-              format: /\Ahttps:\/\/(www.behance.net|behance.net)\/([a-zA-Z0-9\-\_]{3,20})\/?\Z/
+              format: /\A(http(s)?:\/\/)?(www.behance.net|behance.net)\/.*\Z/
   validates :linkedin_url,
               allow_blank: true,
               format:
-                /\Ahttps:\/\/(www.linkedin.com|linkedin.com|[A-Za-z]{2}.linkedin.com)\/in\/([a-zA-Z0-9\-]{3,100})\/?\Z/
+                /\A(http(s)?:\/\/)?(www.linkedin.com|linkedin.com|[A-Za-z]{2}.linkedin.com)\/.*\Z/
   validates :dribbble_url,
               allow_blank: true,
-              format: /\Ahttps:\/\/(www.dribbble.com|dribbble.com)\/([a-zA-Z0-9\-\_]{2,20})\/?\Z/
+              format: /\A(http(s)?:\/\/)?(www.dribbble.com|dribbble.com)\/.*\Z/
   validates :medium_url,
               allow_blank: true,
-              format: /\Ahttps:\/\/(www.medium.com|medium.com)\/([a-zA-Z0-9\-\_\@\.]{2,32})\/?\Z/
+              format: /\A(h(ttp(s)?:\/\/)?)(www.medium.com|medium.com)\/.*\Z/
```

Comment:
Why is this `(h(ttp(s)?:\/\/)?)` different from the others?

Reply:
Oh not sure. I think that's a typo and then somehow I corrected the typo.

Creating a suggestion for myself to fix via GitHub:
```suggestion
              format: /\A(http(s)?:\/\/)?(www.medium.com|medium.com)\/.*\Z/
```